tests/integration/tables/yara.cpp

```diff
@@ -22,26 +22,24 @@ class yara : public testing::Test {
 };
 
 TEST_F(yara, test_sanity) {
-  // 1. Query data
-  auto const data = execute_query("select * from yara where path = ''");
-  // 2. Check size before validation
-  // ASSERT_GE(data.size(), 0ul);
-  // ASSERT_EQ(data.size(), 1ul);
-  // ASSERT_EQ(data.size(), 0ul);
-  // 3. Build validation map
-  // See helper.h for avaialbe flags
-  // Or use custom DataCheck object
-  // ValidationMap row_map = {
-  //      {"path", NormalType}
-  //      {"matches", NormalType}
-  //      {"count", IntType}
-  //      {"sig_group", NormalType}
-  //      {"sigfile", NormalType}
-  //      {"strings", NormalType}
-  //      {"tags", NormalType}
-  //}
-  // 4. Perform validation
-  // validate_rows(data, row_map);
+  // Query data from yara table
+#if !defined(WINDOWS)
```

Comment:
Why is windows excluded? (Windows has yara support since https://github.com/osquery/osquery/pull/6564)

Reply:
Thanks, @directionless for the review. I encounter `SEH` exception while running the integration test on Windows. Although I don't encounter any issue running the same query with `osqueryi`. 

It will be great if you could give feedback about resolving this issue. 

```
unknown file : error : SEH exception with code 0xc0000005 thrown in the test body.
```

Reply:
I have no idea what that error means. But I would interpret it to mean there's a problem with the table or the test.

Reply:
This is interesting, do you mind attaching a debugger and understanding more why this exception is thrown?

Reply:
The Yara library was not getting initialized with the integration test and causing the SEH exception. It was getting done inside `YARAConfigParserPlugin::setUp` function. I add function to initialize/finalize the yara library, if not done previously.

Reply:
The windows build seems to be failing while installing the test prerequisites.